web/client/utils/Filter/converters/index.js

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
just a note: currently we are not using uppercase names for folder in the utils for other files

![image (4)](https://user-images.githubusercontent.com/19175505/218689365-d88423c7-8e4a-495d-9972-21e944b565d2.png)

Reply:
going to fix